bank/services/stoneBaas/stoneBaas.go

```diff
@@ -80,94 +82,40 @@ func (b bankStoneBaas) RegisterBoleto(boletoRequest *models.BoletoRequest) (mode
 
 	b.log.Response(res, baasUrl, mapResponseHeaders(resHeaders))
 
-	return models.BoletoResponse{
-		StatusCode: statusCode,
-	}, err
+	return mapStoneBaasResponse(res, statusCode, err), nil
```

Comment:
Opa, como já estamos no serviço da StoneBaas, talvez dê para simplificar o nome 😀 
```suggestion
	return mapResponse(res, statusCode, err), nil
```